lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -15,6 +15,8 @@ foam.CLASS({
     'org.chromium.webidl.URLExtractor',
   ],
 
+  imports: [ 'outputDAO?' ],
```

Comment:
Do we expect this to be configured by something other than this component's owner? Usually I'm happy with configure-by-context, but since this is doing `this.optionalThing && ...`, the console will get spammed with warnings if `outputDAO` is missing (that is, it will in dev mode, which is pretty much the only mode for this bot code).